cycode/cli/commands/scan/code_scanner.py

```diff
@@ -148,14 +151,21 @@ def _scan_batch_thread_func(batch: List[Document]) -> Tuple[str, CliError, Local
 
         scan_id = str(_generate_unique_id())
         scan_completed = False
+        should_use_scan_service = _should_use_scan_service(scan_type, scan_parameters)
 
         try:
             logger.debug('Preparing local files, %s', {'batch_size': len(batch)})
             zipped_documents = zip_documents(scan_type, batch)
             zip_file_size = zipped_documents.size
-
             scan_result = perform_scan(
```

Comment:
ig it is possible to move

`should_use_scan_service = _should_use_scan_service(scan_type, scan_parameters)`

inside of `perform_scan` function without additional args (because it's already too much)

and for debug logs `scan_type` and `scan_parameters` are enough to understand what is going on